### PR TITLE
[DC] Fixed Cisco Umbrella Connector organization_id parsing

### PR DESCRIPTION
**Pull request by alldoami**

### CODE REVIEW

File: src/connectors/cisco_umbrella.py [hunk 1]
```diff
@@ -104,7 +105,6 @@ def ingest(table_name, options):
     landing_table = f'data.{table_name}'
     timestamp = datetime.utcnow()
 
-    organizations_id = options['organization_id']
```
Comment (edulop91) [inline src/connectors/cisco_umbrella.py]: @alldoami  we might still need this

File: src/connectors/cisco_umbrella.py [hunk 2]
```diff
@@ -85,6 +85,7 @@ def get_data(organization_id: str, key: str, secret: str, params: dict = {}) ->
 def connect(connection_name, options):
     table_name = f'cisco_umbrella_{connection_name}_connection'
     landing_table = f'data.{table_name}'
+    options['organization_id'] = int(options['organization_id'])
```
Comment (edulop91) [inline src/connectors/cisco_umbrella.py]: :+1:

Comment (greg-snowflake) [reply src/connectors/cisco_umbrella.py]: Does this break backwards compatibility with a connector set up before this patch? Or do the options get treated the same once they get yaml-ized and stored in the table comment?

Just double checking.

Comment (andrey-snowflake) [reply src/connectors/cisco_umbrella.py]: yaml treats strings and int differently, just like JSON does, so the comments in currently deployed connectors should be updated. don't think anyone else is using this one just yet, but in future we should include this in migration notes.

File: src/connectors/cisco_umbrella.py [hunk 3]
```diff
@@ -33,7 +33,7 @@
         'name': 'organization_id',
         'title': "Cisco Umbrella Organization Id",
         'prompt': "Your Cisco Umbrella Organization Id",
-        'type': 'str',
+        'type': 'int',
```
Comment (andrey-snowflake) [inline src/connectors/cisco_umbrella.py]: going to add this to the general runner here as we do with JSON: https://github.com/chanzuckerberg/SnowAlert/blob/v1.8.6/src/runners/connectors_runner.py#L43-L44

**Comment by andrey-snowflake** [comment]
re-created under #301

**Pull request closed without merging**